package.json

```diff
@@ -39,6 +39,8 @@
     "eslint": "^3.14.0",
     "eslint-config-airbnb-base": "^11.0.1",
     "eslint-import-resolver-webpack": "^0.8.1",
+    "chart.js": "^2.4.0",
+    "dat.gui": "^0.6.1",
```

Comment:
And that what I answered to.

> dat.gui is used in GuiTools that does not use webpack

GuiTools is not processed. Therefore, pure javascript only. So no you can't.